src/signals/settings/users/containers/Overview/hooks/useFetchUsers.js

```diff
@@ -21,7 +21,7 @@ const useFetchUsers = ({ page, pageSize } = {}) => {
     const controller = new AbortController();
     const { signal } = controller;
 
-    async function fetchData() {
+    (async function fetchData() {
```

Comment:
Name `fetchData` no longer needed. also wondering on performance of creating an anonymous function every time.

Reply:
The function is only recreated when either `page` or `pageSize` changes, so that's intended

Reply:
Also, linter complains about unexpected unnamed function